python/lsst/pipe/tasks/healSparseMapping.py

```diff
@@ -262,23 +267,20 @@ def mask_warp_bbox(self, bbox, visit, mask, bit_mask_value):
                                                     degrees=True)
         bad_hpix = hpg.angle_to_pixel(self.config.nside, bad_ra, bad_dec)
 
-        # Count the number of bad image pixels in each healpix pixel
-        min_bad_hpix = bad_hpix.min()
-        bad_hpix_count = np.zeros(bad_hpix.max() - min_bad_hpix + 1, dtype=np.int32)
-        np.add.at(bad_hpix_count, bad_hpix - min_bad_hpix, 1)
-
-        # Add these to the accumulator map.
-        # We need to make sure that the "primary" array has valid values for
-        # this pixel to be registered in the accumulator map.
-        pix_to_add, = np.where(bad_hpix_count > 0)
-        count_map_arr = self._ccd_input_bad_count_map[min_bad_hpix + pix_to_add]
-        primary = self._ccd_input_bad_count_map.primary
-        count_map_arr[primary] = np.clip(count_map_arr[primary], 0, None)
+        # Check if any of these "bad" pixels are in the valid footprint.
+        match_input, match_bad = esutil.numpy_util.match(self._ccd_input_pixels, bad_hpix, presorted=True)
```

Comment:
There's no possible way  `self._ccd_input_pixels` wouldn't be sorted right?

Reply:
It should be impossible with the way we are making the healsparse map, but I have added an explicit sorting step when they are made just in case.